frontend/src/store/socket/index.js

```diff
@@ -92,34 +94,57 @@ export const useSocketStore = defineStore('socket', () => {
     socket.disconnect()
   }
 
-  function emitSubscribe (options) {
-    if (socket.connected) {
-      socket.emit('subscribe', 'shoots', options, ({ statusCode, message }) => {
-        if (statusCode === 200) {
-          logger.debug('subscribed shoots')
-          shootStore.setSubscriptionState(constants.OPEN)
-        } else {
-          const err = new Error(message)
-          err.name = 'SubscribeError'
-          logger.debug('failed to subscribe shoots: %s', err.message)
-          shootStore.setSubscriptionError(err)
-        }
+  async function emitSubscribe (options) {
+    if (!socket.connected) {
+      return
+    }
+    const {
+      statusCode = 500,
+      message = 'Failed to subscribe shoots',
+    } = await socket.timeout(5000).emitWithAck('subscribe', 'shoots', options)
```

Comment:
Increased the timeout to 60 sec. The loading indicator of the table is now limited to the loading of initial list not the subscribing to the channel. This is indicated only in the connection status icon in the toolbar. It has always been the concept that in case of errors the retry must be done manually. Subscription errors are logged now.